tests/test_orm.py

```diff
@@ -64,20 +65,24 @@ class Meta:
         email="gui@gui.com",
         is_registered=True,
         birthday=datetime(2020, 12, 12).date(),
+        photo="{json: photo}"
```

Comment:
what is this line meant to? It seems like invalid json

Reply:
It's filler text for a read-only field. Syntax shouldn't matter.

Reply:
as it is confusing, I'd set it to something less confusing like "Test Text" etc.